scale_build/bootstrap/configure.py

```diff
@@ -0,0 +1,101 @@
+import os
+import shutil
+
+from scale_build.exceptions import CallError
+
+from .cache import check_basechroot_changed, create_basehash, save_build_cache, validate_basecache
+from .cleanup import remove_boostrap_directory
+from .logger import get_logger
+from .utils import BUILDER_DIR, CACHE_DIR, CHROOT_BASEDIR, get_apt_preferences, get_manifest, run
+
+
+def make_bootstrapdir(bootstrapdir_type, logger_file=None):
+    assert bootstrapdir_type in ('cdrom', 'package')
+    remove_boostrap_directory()
+    try:
+        _make_bootstrapdir_impl(bootstrapdir_type, logger_file)
+    finally:
+        remove_boostrap_directory()
```

Comment:
Doesn't this function really make a cached bootstrap image? It seems weird that `make_something` calls `remove_something` at the very end? Or are `bootstrapdir` and `boostrap_directory` different things?